build/musicbrainz-dev/scripts/fetch-dump.sh

```diff
@@ -98,18 +98,37 @@ then
 	exit 64 # EX_USAGE
 fi
 
+# Keep support for (deprecated) FTP option (which still takes precedence)
+
+BASE_DOWNLOAD_URL="${BASE_FTP_URL:-$BASE_DOWNLOAD_URL}"
+
 # Fetch latest search indexes
 
 if [[ $TARGET =~ ^(both|search)$ ]]
 then
 	echo "$(date): Fetching search indexes dump..."
-	cd "$SEARCH_DUMP_DIR" && find . -delete && cd -
+	PREVIOUS_DUMP_TIMESTAMP=$(
+		if [[ -a "$SEARCH_DUMP_DIR/LATEST" ]]
+		then
+			cat "$SEARCH_DUMP_DIR/LATEST"
+		fi
+	)
```

Comment:
Not a fan of control structures inside sub-shells:

```suggestion
	PREVIOUS_DUMP_TIMESTAMP=""
	if [[ -a "$SEARCH_DUMP_DIR/LATEST" ]]
	then
		PREVIOUS_DUMP_TIMESTAMP=$(<"$SEARCH_DUMP_DIR/LATEST")
	fi
```

Reply:
Changed for all occurrences in commit https://github.com/metabrainz/musicbrainz-docker/pull/245/commits/e3a6cf9d8855819d68e4c2b4891a10355c58410e